### DISCUSSION TITLE
🌼 Decorations & Landscaping

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#2195] [Ideas]
The game is known for its beautiful colourful crops, beautiful buildings and charming decorations.

As we expand, we plan to introduce more diverse decorations and empower the players to organise their islands. This brings on a new gameplay element & motivation in the game - make your island beautiful!  As real-time chat is enabled, you will want the best island so the Bumpkins come and visit you.

The purpose of this discussion is to explore the mechanics that will make it possible. We are also looking for decoration ideas from the player base! 🤗 ❤️ 

### Landscaping

The first stage of this proposal enable players to dig up and move resources with a Power Shovel.

The Power Shovel will be a one-time purchase and can be used infinitely. With the latest [Farming Equipment](https://github.com/sunflower-land/sunflower-land/discussions/2148) proposal, area of impact now greatly affects your farming abilities. Hence, the Power Shovel will be an expensive and difficult tool to attain, reserved for later levels.

With a Power Shovel, you can either remove a resource node or move it elsewhere. If you decide to remove it, it will stay in your Chest until placed - resource nodes are non-tradable.

_Technical Note_

As part of this initiative, we will be adding resources (trees + rocks) to our base ERC1155 Inventory contract. This will enable Open Source developers to query how many tree or rock nodes a player has.

**Land Scarcity**

With the ability to move resource nodes, land will be freed up and will be easier to place necessary buildings & equipment. This means a player could require less expansions to place more buildings. To ensure our core game loop (expanding) continues as intended, we will require certain buildings require X pieces of land.

For instance, you must have 6 pieces of land to place a Chicken Coop.

This aims to deter players who want to squeeze every possible building on just the initial land layout. 

### Rusty Shovel

We want to give the players the freedom to reorganise their farm when they want and however they want. Given that players will be making many more changes, we will be updating the Rusty Shovel to be a one-time purchase that can be used infinitely. This will involve an increase in resources/SFL to craft it. Prices _TBD_

### Decorations

The team is planning to add the following decorations for purchase with SFL and/or a combination of resources. There will be no limit on the amount that can be purchased. 

**Wooden Fence**

A wooden fence can be laid out and joined together with other fence pieces to form a long running fence.

**Stone Wall**

A stone wall can be laid out and joined together with other wall pieces to form a long running stone wall.

**Flower Bushes**

A range of white, red and blue flower bushes. These are stand alone decorations for your land.

**Path**

A dirt path that can be laid out and joined together with other dirt pieces to form paths or large areas of dirt.

### More ideas?

Now we are looking for community requests and ideas for what you want to see in the game. We will curate a list and work with the community artists to determine which items to add into the game.

_Examples from Discord chats include_

- Rustic Windmill
- Barrels
- Street Lamp
- Wooden Sign
- Outback toilet
- Picnic Table
- Beach Ball
- Beach Towels
- Beach Chairs
- Tree variations
- Banners
- Swans

Each event we will also include a range of limited edition decorations (Christmas, Valentines etc.)

**Comment by brian-man** [comment]
So far everything above sounds lovely.

**Comment by moksem1337** [comment]
I wanna see some garden gnomes with different coloured hats!

**Reply by adamhannigan** [reply]
Love it!

We currently have a Gnome SFT. Perhaps we could create a 3 tiered structure:

- Basic Blue Hat Gnome
- Advanced Green Hat Gnome
- Legendary Red Hat Gnome (current SFT)

**Reply by therocco** [reply]
@adamhannigan Shouldn't legendary be sunflower yellow? :grin:

**Comment by vitt0** [comment]
Different types of land+trees sound like something cool. Maybe they can be "seasonal", like they would be available for a limited time in Helios but the players that got them can use it as long they want.

Like for example a desert theme, that makes the land as sand and the trees are palm, just in the same way that we have the Christmas event, where farms were with snow and the trees too 😄 

Edit: This "themes of landscape" could be just 1 NFT that after is placed in our farm it changes all the land colors + trees

**Comment by alphacruze** [comment]
pathing, like a dirt road kind of thing

**Comment by Ogrekin** [comment]
Looks good so far first thing that came in my mind was windmill but that one was already mentioned and that is great that one
is a must.
I would like to see some kind of an old silo and maybe an "Texas" oil rig...
A pond or similar that has a farm style or maybe even a "shit tank" for cow manure...
I would like my farm to look more like an cool old cowboy farm...
Just some of thought that pop up will scout internet for cool items but something with cowboys had been cool to...

**Comment by Shuun7** [comment]
I think a little brown square would be nice, meant to draw the way into the land. Each player will be able to draw a path between all the buildings on his land, by minting out a large number of these squares and connecting them. I hope you understand my meaning..

**Comment by Jakesmoked-141829** [comment]
Butterfly and   ...  snails comeback ?? =)

**Comment by Rylinn** [comment]
Love it!! Can not wait for more and I hope I can help out with one asset as artist:)

**Comment by ReC82** [comment]
What about : 

- a flag with our farm id on it ?
- A statue of Lilly ? Or/And the Homeless guy ?
- A statue of the COVID (well, bad idea, but in many years, maybe - we know where we come from :p)
- A statue for each crops (like the potato)

And i'll suggest a nft that would cost something real like 10 or 50 SFL that is related to support you, the devs, not extractrable, of course, no boost, but where the most part of the "money" goes to you guys.  Think of it as a tip, but where people get something of it and can sayr proudly on his farm, we support the devs.  I would definetvely buy one ;).

**Reply by ReC82** [reply]
And imo, the best you could do is decoration for "real" money, I mean SFL, Matic, or what ever you want, to support real causes.  Let's say that you choose a organisation which save dolphins, create a flag that would cost a thing like 5 SFL and we can place it on our Island, or better, on our personnal charity Island :)

**Reply by DeccoMx** [reply]
I like really much the Flag ID idea, and adding to your proposal, i think it could be awesome if we got multiples ways to show our farm ID:
In the form of a flag, wooden signs, stone carved, etc. 

And maybe there could be like a bulletin board item with a paper that say something like "Lockdown" or "Keep your dastans", referring to the COVID days. Cous i think a COVID statue could be too much xD.

**Comment by TCoutz** [comment]
I know that this is not the subject here, but you could create NFTs that synergize with decorations. For instance, bees could produce more near flower pots. Or fenced animals have some kind of boost. This could create utility to decorations and encourage players to make their farm both beautiful and functional.

**Reply by ReC82** [reply]
If I read well the roadmap, it's kind of already the case.  But, imo, decoration, is what is it si decoration.   You can put a boost as decoration, but a decoration should have its own purpose, decoration :).

**Comment by Factor006** [comment]
hello, everything looks beautiful. thanks. I have a couple of suggestions as well. 
Camera 
souvenir album 
camp Fire swing bird's nest 🐦 
These are the ones that come to mind for now. +++ Decoration materials exp. supplementation can be provided. For example, it would be nice for the bathroom to give +exp 👍

**Comment by ShiningMinions** [comment]
Everything in this proposal sounds really good to me! :clap: 

> Given that players will be making many more changes, we will be updating the Rusty Shovel to be a one-time purchase that can be used infinitely.

Love it :heart: . I think having some of these one-time purchase tools also would be great contenders for the Helios blacksmith (tools that don't expire). This gives a powerful single-player experience and appeases to audiences who don't care about competition and/or community involvement. This could also be a fantastic place for the regular shovel (when it gets re-enabled). This would also give early players even more fun things to grind for (= more things to look forward to = more fun).

You can also consider maybe adding another one-time tool: "hammer". I know we can place buildings today without a tool, but perhaps grinding for your "hammer" in order to place your first buildings could be an interesting idea for new players (and make it more fun).

I'm obsessed for finding a use-case for the "hammer" :smile: .

**Comment by cryptokohai** [comment]
It has been mentioned that each land itself is an NFT that can be traded. 

How will this reconcile with the fact that different lands cost different resources to mint and that resources on the land cannot be traded but the land itself can?

**Reply by brian-man** [reply]
> It has been mentioned that each land itself is an NFT that can be traded. How will this reconcile with the fact that different lands cost different resources to mint and that resources on the land cant be traded but the lang itself can?

Maybe put some thoughts here:
https://github.com/sunflower-land/sunflower-land/discussions/2020 "Expansions" as tradable NFTs

Hopefully @adamhannigan will take a look.

**Comment by Alaindrea** [comment]
how about cobblestone path and flamingos

**Comment by Beni25fps** [comment]
A "Rooster direction sign"
The sign can be made of various materials such as wood, stone, or Gold,
and can be designed in different styles and sizes

**Comment by SarielWang93** [comment]
When can shoveled fruit, and shoveled crops resume?

**Comment by ghost** [comment]
It will be very cool if the color of the roof of the building can be changed.

**Comment by FarmerKuenster** [comment]
A stone/brick walk way or path
Garden trellis/A-frame that could fit over the path
free standing "porch" swing

**Reply by therocco** [reply]
I would absolutely love the ability to create different types of paths out of different material at the workbench, especially if movement is allowed on player islands like we have seen in the Plaza demos.

**Comment by CosmicMinx** [comment]
The decorations are wonderful. But having no reason to show it off (other than screenshots) is not fun. I would like to have a reason to visit other farms. Maybe a sign that says "the big seed" that I could earn and place on my land, and then players that visit can click that sign and get some sort of a collection reward once a week. Deco needs to be shown off! The reward would be related to a collection. Collect objects by clicking on neighbors signs when visiting and then earn some sort of reward when complete. It would allow players to work on earning the sign first, then placing it and then listing it so others can visit and work towards the collection. Could be as simple as a title in game that is the reward. Or, you can change it up every few months. An ongoing object of usefulness :)

**Comment by Alaindrea** [comment]
How about a wooden  bench that the bumpkins can actually sit on? A beach umbrella? a Waterslide? I want the waterslide very bad.

**Comment by Rylinn** [comment]
Wooden Fenc, Rustic Windmill are my favorite already! Sounds good.
I would love to see a pond with ducks. Maybe wood signs painted with a depending vegtable that can be placed next to crops. Like the ideas above

**Comment by brian-man** [comment]
Please make sure node placement and node move rules align with the "birth" rules for nodes when expanding.

More specifically, trees are allowed to use the triangle corners at birth while building and collectible placement currently do not allow that.

![image](https://user-images.githubusercontent.com/36871683/218548795-18472940-1105-4881-b2d7-ac70f783bd82.png)

**Comment by DeshFarm** [comment]
Acho que isso entrar o quanto antes é essencial, pois muitas fazendas tem uma visão muito poluída devido aos campos e recursos gerarem em lugares aleatórios se esse recurso de reorganizar entrar no jogo, irá fazer com que as fazendas tenham uma nova identidade e uma nova cara que agrada cada vez mais os jogadores.

I think that entering as soon as possible is essential because many farms have a very polluted one due to the fields and resources generating in random places, if this reorganizing feature enters the game, it will make the farms have a new identity and a new face which pleases players more and more.

**Reply by adamhannigan** [reply]
_Concordo! Tem muitas fazendas feias agora!_

Definitely, this is something I am pushing for. I hate seeing all these disorganized farms 😅

**Comment by Prad0visk** [comment]
please, add day and night time, woud give farms and " lamps" a good worth

**Comment by On1el** [comment]
![1111](https://user-images.githubusercontent.com/104461938/221229806-858db0df-c243-49e5-810a-49d14887e0b3.png)
**I wish we had the ability to expand the SFT bears horizontally**

**Reply by ghost** [reply]
I think such a function can be added for many NFTs
